tests/reference/run_dbg-test_assert_01-2f34744.stderr

```diff
@@ -1,5 +1,2 @@
-  File "tests/runtime_errors/test_assert_01.py", line 4
-    test()
-  File "tests/runtime_errors/test_assert_01.py", line 2
-    assert False
+warning: The `runtime stacktrace` is not enabled. To get the stacktraces, re-build LPython with `-DWITH_RUNTIME_STACKTRACE=yes`
```

Comment:
Hi, are you using `MSVC `on a windows device to build ?

Reply:
I doubt that this change should not happen here. You should run `python run_tests.py -u --skip-run-with-dbg`